src/k8s/cmd/k8s/k8s.go

```diff
@@ -4,11 +4,19 @@ import (
 	"time"
 
 	cmdutil "github.com/canonical/k8s/cmd/util"
+	"github.com/canonical/k8s/pkg/k8sd/features"
 	"github.com/spf13/cobra"
 )
 
 var (
-	featureList = []string{"network", "dns", "gateway", "ingress", "local-storage", "load-balancer"}
+	featureList = []string{
+		string(features.Network),
+		string(features.DNS),
+		string(features.Gateway),
+		string(features.Ingress),
+		string(features.LocalStorage),
+		string(features.LoadBalancer),
+	}
```

Comment:
While we're at it, this could also be in the features package, maybe something like a `func All() []types.FeatureName` used as `features.All()`?

Reply:
there was some old discussion, maybe `metrics-server` would not be in that list

Reply:
If we want to exclude `metrics-server` (so that we can use it here) the `features.All()` function would be somehow misleading. maybe something like `func All(includeMetricsServer bool) []types.FeatureName` would cover that but I don't feel good about it (seems kind of ugly). 
Or maybe we can also include the `metrics-server` in this feature list and use the `All()` function.

Reply:
let's do `features.Public()` instead

Reply:
Seems like we have to either go with `func Public() []string` or do something like:
```go
publicFeatures := features.Public()
featuresList := make([]string, len(publicFeatures)
for i, f := range publicFeatures {
    featuresList[i] = string(f)
}
```
cause the `featuresList` is going to be used in `strings.Join` and other places. If I have to choose, I would go with the second option.
wdyt?

Reply:
See https://github.com/canonical/k8s-snap/blob/2e29d066681c32ccc97392ba3ecef8d771454d86/src/k8s/pkg/k8sd/images/images.go#L7-L16

Reply:
in the example you sent it's `var registeredImages []string` and works with `copy`.
https://github.com/canonical/k8s-snap/blob/2e29d066681c32ccc97392ba3ecef8d771454d86/src/k8s/pkg/k8sd/images/images.go#L5